sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -24,35 +24,23 @@
         <Subsystem>CBP</Subsystem>
         <Version>3.8</Version>
         <Author>ecoughlin</Author>
-        <EFDB_Topic>CBP_command_moveAzimuth</EFDB_Topic>
-        <Alias>moveAzimuth</Alias>
+        <EFDB_Topic>CBP_command_move</EFDB_Topic>
+        <Alias>move</Alias>
```

Comment:
Please discard all deprecated fields for CBP. For commands this is Alias, Author, Version, Property, Action, Value, and Explanation. For Events and Telemetry it is a subset of those.